packages/ingest/test/granule.js

```diff
@@ -85,17 +85,6 @@ Object.keys(sums).forEach((key) => {
   });
 });
 
-test('filter files using regex', (t) => {
```

Comment:
Removing this test because the `setGranuleInfo` method is never called outside of this file, and its functionality is covered in other tests.